addons/website_blog/security/ir.model.access.csv

```diff
@@ -2,8 +2,8 @@ id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 blog_blog_all,blog.blog,model_blog_blog,,1,0,0,0
 blog_blog,blog.blog,model_blog_blog,website.group_website_designer,1,1,1,1
 blog_post_all,blog.post,model_blog_post,,1,0,0,0
-blog_post,blog.post,model_blog_post,website.group_website_designer,1,1,1,1
+blog_post,blog.post,model_blog_post,website.group_website_publisher,1,1,1,1

```

Comment:
Do we need to add create/unlink right?
Now restricted editor can create new blog post, not only edit, is that wanted?